astrocut/tests/test_tess_footprint_cutout.py

```diff
@@ -267,3 +268,14 @@ def test_cube_cut_from_footprint(coordinates, cutout_size, tmpdir):
     assert isinstance(cutouts, list)
     assert isinstance(cutouts[0], str)
     assert str(tmpdir) in cutouts[0]
+
+
+def test_get_tess_sectors(coordinates):
+    """Test that get_tess_sectors returns sector list"""
+    sector_table = get_tess_sectors(coordinates, 0)
+    assert isinstance(sector_table, Table)
+    assert 'sectorName' in sector_table.colnames
+    assert 'sector' in sector_table.colnames
+    assert 'camera' in sector_table.colnames
+    assert 'ccd' in sector_table.colnames
+    assert len(sector_table) >= 7
```

Comment:
This only tests cutout_size=0

Can we add some parametrized tests covering:

- cutout_size as int/Quantity/array
- coordinates that match no sectors (are there any?)
- invalid coordinates